WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -264,6 +266,24 @@ private static Intent preparePictureLibraryIntent(String title, boolean multiSel
         return Intent.createChooser(intent, title);
     }
 
+    public static void launchAudioLibrary(Activity activity, boolean multiSelect) {
+        AppLockManager.getInstance().setExtendedTimeout();
+        activity.startActivityForResult(
+            prepareAudioLibraryIntent(activity.getString(R.string.pick_audio), multiSelect),
+            RequestCodes.AUDIO_LIBRARY);
+    }
+
+    private static Intent prepareAudioLibraryIntent(String title, boolean multiSelect) {
+        Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
+        intent.setType(MIME_DATA_TYPE_AUDIO);
+        if (multiSelect) {
```

Comment:
Minor nit: this can be shortened to `if (multiSelect && Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2)`

Reply:
Yup. Fixed.